.circleci/config.yml

```diff
@@ -237,6 +228,19 @@ jobs:
           command: |
             .circleci/helm/test-charts.sh
 
+  cleanup-images:
+    docker:
+      - image: circleci/golang:1.13-stretch
+    steps:
+      - checkout
```

Comment:
Why we need this at all? Should every build to cleanup images produced? I think we need to run it once and at finish of build.

Reply:
Please see #2016 - it has drawbacks performance and security wise